.github/workflows/ci.yml

```diff
@@ -17,20 +16,14 @@ jobs:
     strategy:
       matrix:
         java-version:
-          - 8
           - 11
```

Comment:
I presume testing against 17+ will be added in separate future PRs?

Reply:
Yeah. Right now, the tests fail when Java 17 is used due to intermingled dependencies between the build and testing.